### PR TITLE
Add notes about volume sizes before Pulp3 migration

### PR DESCRIPTION
**Pull request by lzap**
2.5 only version of https://github.com/theforeman/foreman-documentation/pull/883/files

@jlsherrill

### CODE REVIEW

File: guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc [hunk 1]
```diff
@@ -6,6 +6,15 @@ For large systems, this can mean several days of downtime.
 To prevent this, pre-migrate Pulp content while running the latest version of {ProjectServer} {ProjectVersionPrevious}.
 This reduces the overall upgrade downtime.
 
+During migration to Pulp3, amount of data stored in the `/var/lib/pulp/published/` directory can be doubled.
```
Comment (melcorr) [inline guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc:9]:
```suggestion
During migration to Pulp3, the amount of data stored in the `/var/lib/pulp/published/` directory can double in size.
```

File: guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc [hunk 2]
```diff
@@ -6,6 +6,15 @@ For large systems, this can mean several days of downtime.
...
+Ensure there is enough space on the `/var/lib/pulp` volume.
```
Comment (melcorr) [inline guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc:10]:
```suggestion
Ensure that there is enough space on the `/var/lib/pulp` volume.
```
```suggestion
Ensure there is enough space on the `/var/lib/pulp` volume.
```

File: guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc [hunk 3]
```diff
@@ -6,6 +6,15 @@ For large systems, this can mean several days of downtime.
...
+If the user has run `{foreman-maintain} prep-6.10-upgrade`, the content in `/var/lib/pulp/content` should not need to be duplicated.
```
Comment (melcorr) [inline guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc:11]:
```suggestion
If the user runs `{foreman-maintain} prep-6.10-upgrade`, the content in `/var/lib/pulp/content` does not need to be duplicated.
```

File: guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc [hunk 4]
```diff
@@ -6,6 +6,15 @@ For large systems, this can mean several days of downtime.
...
+After Pulp2 is removed, the extra space can be claimed back, however, shrinking of XFS volumes is not possible and copying data to a different partition might be necessary in order to bring volume size down to the previous value.
```
Comment (melcorr) [inline guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc:12]:
```suggestion
After Pulp2 is removed, the extra space can be claimed back, however, shrinking of XFS volumes is not possible and copying data to a different partition might be necessary in order to bring volume size down to the previous value.
```

File: guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc [hunk 5]
```diff
@@ -6,6 +6,15 @@ For large systems, this can mean several days of downtime.
...
+
+During migration to Pulp3, amount of data stored in the PostgreSQL data directory can grow significantly.
```
Comment (melcorr) [inline guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc:14]:
```suggestion
During migration to Pulp 3, the amount of data stored in the PostgreSQL data directory can grow significantly.
```

File: guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc [hunk 6]
```diff
@@ -6,6 +6,15 @@ For large systems, this can mean several days of downtime.
...
+Make sure to have enough space for the migration.
```
Comment (melcorr) [inline guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc:15]:
```suggestion
Ensure that you have enough space for the migration.
```

File: guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc [hunk 7]
```diff
@@ -6,6 +6,15 @@ For large systems, this can mean several days of downtime.
...
+Rough estimation is 2-3 times the size of the MongoDB Pulp2 database.
```
Comment (melcorr) [inline guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc:16]:
```suggestion
A rough estimation of the space you require is 2-3 times the size of the MongoDB Pulp 2 database.
```

**Review by melcorr — CHANGES_REQUESTED** [review]

**Comment by ekohl** [comment]
I'm not sure if this is the right place but something that was also noted by a user on IRC just now: it will also install PostgreSQL on Capsules now. In Satellite 6.9 they should think about their partitioning. It will create a PostgreSQL DB in `/var/opt/rh/rh-postgresql12` that will grow. If the user wants to store this on a separate partition, it would be good to create this as otherwise their `/` or `/var` may grow in an unexpected way.

**Comment by lzap** [comment]
Merged all your corrections @melcorr please squash.

@ekohl not feeling competent to do such a change, but will happily review if you go for it. I have a 6.10 to test with.

**Pull request merged**